libmambapy/src/main.cpp

```diff
@@ -578,6 +577,7 @@ PYBIND11_MODULE(bindings, m)
         .def_readwrite("retry_backoff", &Context::RemoteFetchParams::retry_backoff)
         .def_readwrite("user_agent", &Context::RemoteFetchParams::user_agent)
         // .def_readwrite("read_timeout_secs", &Context::RemoteFetchParams::read_timeout_secs)
+        .def_readwrite("proxy_servers", &Context::RemoteFetchParams::proxy_servers)
```

Comment:
Btw, if we still want backward compatibility with python API, we need to add something like this:
https://github.com/mamba-org/mamba/blob/main/libmambapy/src/main.cpp#L605-L621